compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -573,10 +573,12 @@ impl<'local, 'interner> Interpreter<'local, 'interner> {
     }
 
     pub(super) fn evaluate_ident(&mut self, ident: HirIdent, id: ExprId) -> IResult<Value> {
-        let definition = self.elaborator.interner.try_definition(ident.id).ok_or_else(|| {
-            let location = self.elaborator.interner.expr_location(&id);
-            InterpreterError::VariableNotInScope { location }
-        })?;
+        let location = self.elaborator.interner.expr_location(&id);
```

Comment:
It looks like this `location` only needs to be looked-up in the `ok_or_else` case below?

Reply:
It's also used on lines 672 and another one. But I'll push a commit to only fetch it when needed.